docs/Using-Docker.md

```diff
@@ -26,13 +27,16 @@ Since Docker typically runs a container sharing a (linux) kernel with the host m
 Unity environment **has** to be built for the **linux platform**. When building a Unity environment, please select the following options from the the Build Settings window:
 - Set the _Target Platform_ to `Linux`
 - Set the _Architecture_ to `x86_64`
-- `Uncheck` the _Development Build_ option
-- `Check` the _Headless Mode_ option. (_This is required because the Unity binary will run in a container that does not have graphics drivers installed_.) 
 
-![Build Settings For Docker](images/docker_build_settings.png)
+![Build Settings For Docker](images/docker_build_settings_noheadless.png)
 
 Then click `Build`, pick an environment name (e.g. `3DBall`) and set the output directory to `unity-volume`. After building, ensure that the file `<environment-name>.x86_64` and subdirectory `<environment-name>_Data/` are created under `unity-volume`.
 
+**NOTE** If you are only collecting vector observations from Unity, you can select the `headless` option here:
```

Comment:
Consider including the headless commentary in the list above. That why when a user is following the instructions, they better understand what role the headless option represents.

Reply:
"If you are only collecting vector observations from Unity" --> "If the environment does not contains visual observations"

Reason for this change too is that headless presumably is about visual, so if they're using vector and/or text observations then they should be ok.

Reply:
So I am going to fold the `NOTE` into the checklist. Let me know if you think the two images for the `headless` and `noheadless` options seem useless now?